bittensor/synapses/bert.py

```diff
@@ -6,6 +6,32 @@
 import transformers
 from transformers import BertModel, BertConfig
 
+def mlm_batch(data, batch_size, tokenizer, collator):
```

Comment:
This is a good idea, moving it out of the specific example, however, we should create a new dir for dataloaders. This is probably to big for this commit but we will want them to be abstracted and easy to import. 

tldr: mlm_batch is not bert specfic

Reply:
Good point! I'll make the new dir next commit.